### PR TITLE
[release] v0.0.1-alpha

### PR DESCRIPTION
**Pull request by begizi**

### CODE REVIEW

File: release/.env.sample [hunk 1]
```diff
@@ -0,0 +1,7 @@
+GIN_MODE=release
+PORT=8888
+PUBLIC=client
+GITHUB_KEY=
+GITHUB_SECRET=
+SLACK_API=
```
Comment (samccone) [inline release/.env.sample]: :goat: ?

**Comment by samccone** [comment]
do we want to release to a branch? and then just push the branch to the :pie:

File: release/.env.sample (hunk 1 above)
Comment (begizi) [inline release/.env.sample]: Oops. Making PR to change sample port and public folder to more sensible defaults.

**Comment by begizi** [comment]
@samccone Wouldn't it be a pain to keep the branch up to date with master? @joelhawksley What do you think?

**Comment by samccone** [comment]
maybe we are talking about different things here, I am talking about "deploy"

like `make deploy` would compile ... push to the deploy branch ... push that to the pi ... and restart

what do you mean by "release" here?

**Comment by begizi** [comment]
Ah I see. So we wouldn't push the build assets to the github remote, just the pi remote. That would work better.

I still think we should do releases to allow others to download the run the pi binary without having to compile; we can just add that to the downloads section of the repo.

**Pull request closed without merging**